### PR TITLE
FR47180 Holistic instrument tracking folders - outlier level setting ability

### PR DESCRIPTION
**Pull request by ankurjuneja**
#### Rationale
This work provides the ability to configure different outlier level settings for different metrics.

#### Related Pull Requests
* 

#### Changes
* change targetedms.QCEnabledMetric.enabled to targetedms.QCEnabledMetric.status
* use lowerBound and upperBound columns to define outlier level settings
* change plot type name from 'Levey-Jennings' to 'Value'

### CODE REVIEW

**Comment by ankurjuneja** [comment]
@labkey-jeckels these branches are ready for review.

File: resources/views/configureQCMetric.html [hunk 1]
```diff
@@ -20,29 +20,40 @@
 
             qcMetricsTable += '<table class="labkey-data-region-legacy labkey-show-borders">' +
                                 '<tr>' +
-                                    '<td class="labkey-column-header">Status</td>' +
                                     '<td class="labkey-column-header">Name</td>' +
                                     '<td class="labkey-column-header">Type</td>' +
+                                    '<td class="labkey-column-header">Metric Usage</td>' +
```
Comment (labkey-jeckels) [inline resources/views/configureQCMetric.html]: Consistent capitalization.

```suggestion
                                    '<td class="labkey-column-header">Metric usage</td>' +
```

File: test/src/org/labkey/test/tests/panoramapremium/TargetedMSQCPremiumTest.java [hunk 2]
```diff
@@ -129,7 +130,7 @@ public void testNotification()
         assertEquals("Mismatch in the expected number of notification", 1, notifications.getEmailCount());
         EmailRecordTable.EmailMessage message = notifications.getMessageWithSubjectContaining("Panorama QC Notification");
         notifications.clickMessage(message);
-        assertTextPresent("38 outliers"); //Total outliers
+        assertTextPresent("0 outliers"); //Total outliers
```
Comment (labkey-jeckels) [inline test/src/org/labkey/test/tests/panoramapremium/TargetedMSQCPremiumTest.java:133]: This seems odd to me, but I guess maybe all the outliers in the previous version were CUSUM and Moving Range?

File: webapp/TargetedMS/js/QCSummaryPanel.js [hunk 3]
```diff
@@ -369,55 +362,55 @@ Ext4.define('LABKEY.targetedms.QCSummary', {
...
                 else {
-                    content += '<td align="right">' + this.getSampleDetailOutlierDisplayValue(item, 'LeveyJennings') + '</td>';
-                    content += '<td align="right">' + this.getSampleDetailOutlierDisplayValue(item, 'mR') + '</td>';
-                    content += '<td align="right">' + this.getSampleDetailOutlierDisplayValue(item, 'CUSUMmN') + '</td>';
-                    content += '<td align="right">' + this.getSampleDetailOutlierDisplayValue(item, 'CUSUMmP') + '</td>';
-                    content += '<td align="right">' + this.getSampleDetailOutlierDisplayValue(item, 'CUSUMvN') + '</td>';
-                    content += '<td align="right">' + this.getSampleDetailOutlierDisplayValue(item, 'CUSUMvP') + '</td>';
-                    content += '<td align="right">' + this.getSampleDetailOutlierDisplayValue(item, 'Total') + '</td>';
+                    content += '<td style="text-align: right">' +
+                            (item.MetricStatus === 'PlotOnly' ? 'N/A' : this.getSampleDetailOutlierDisplayValue(item, 'Value')) +
```
Comment (labkey-jeckels) [inline webapp/TargetedMS/js/QCSummaryPanel.js]:
```suggestion
                            (item.MetricStatus === LABKEY.targetedms.MetricStatus.PlotOnly ? 'N/A' : this.getSampleDetailOutlierDisplayValue(item, 'Value')) +
```

**Review by labkey-jeckels — APPROVED** [review]
Thanks for all the test updates.

File: test/src/org/labkey/test/tests/panoramapremium/TargetedMSQCPremiumTest.java (hunk 2 above)
Comment (ankurjuneja) [reply test/src/org/labkey/test/tests/panoramapremium/TargetedMSQCPremiumTest.java:133]: Yes, that is correct. I can probably add a comment here.

Comment (labkey-jeckels) [reply test/src/org/labkey/test/tests/panoramapremium/TargetedMSQCPremiumTest.java:133]: That's probably not needed. What caught my eye was the dramatic drop from this change. We don't have comments in other places explaining the outlier counts.

**Pull request merged**